client/src/hooks/useApplicationData.js

```diff
@@ -1,40 +1,35 @@
 import { useEffect, useReducer } from "react";
-import dataReducer, { SET_USERS, SET_DATA } from "../reducer/data_reducer";
+import dataReducer, {
+  SET_STAGE_COLOR,
+  SET_DATA,
+} from "../reducer/data_reducer";
 import axios from "axios";
 import * as d3 from "d3";
 
 const useApplicationData = () => {
   const [state, dispatch] = useReducer(dataReducer, {
-    users: [],
+    stageObj: {},
     mapData: [],
     loading: true,
   });
   useEffect(() => {
-    // get geoJSON from server
-    d3.json("http://localhost:3000/api/maps")
-      .then((data) => {
-        console.log(data);
+    // load geoJSON data and stage # data for each region
+    Promise.all([
+      d3.json("http://localhost:3000/api/maps"),
+      axios.get("http://localhost:3000/api/map_color"),
+    ])
+      .then((response) => {
```

Comment:
you might prefer array-destructuring syntax for Promise.all on line 21:
.then(([mapData,heaalthData])=>{})